rfc/rfc006-distributed-registry.md

```diff
@@ -322,7 +327,66 @@ The care organization refers to it:
 }
 ```
 
-### 4.1. Contact information
+### 4.1 Service references
+
+Any `serviceEndpoint` with a value that starts with `did` is a reference to another service.
+As specified earlier, all references use the `type` query parameter.
+The origin of a reference is from the `serviceEndpoint` context, while the target is a complete `service` object.
+When a reference targets a `service`, it actually targets the `serviceEndpoint` of the service.
+
+A client doesn't care about the references. When a client queries for particular services, the references need to be resolved.
+Resolving a reference might lead to resolving more references and may lead to an infinite loop.
+Resolving a reference MUST NOT go deeper than 5 levels. Given 5 services, A through E, A references B, B references C, etc. Given the maximum depth of 5, E MUST contain an absolute endpoint URI.
```

Comment:
5 sounds like a lot, maybe just start at 3? We can always up it (lowering the limit is much harder)

Given 5 services where `A` ultimately references `E` through `B`, `C` and `D` (`A -> B -> C -> D -> E`)

Reply:
well 3 would be close to a real use-case, so 5 is safe